### ISSUE TITLE
How might I add specific  class before and after clicked on button?

### ISSUE THREAD

**Issue by HelloTalib** [d-e-v-s-k/cuttr-js#7]
HI there!,
Thanks for this awesome library. I have faced some trouble to using this library. I need a specific class on button before and after clicked on button. How can I do this. can you help me?

Regards.
Talib

**Comment by d-e-v-s-k** [comment]
Hi @abutalib4599,
great to hear, that you are using the lib!

Since there are no callback functions implemented yet, you need to do this manually e.g. like this:

``` javascript
let truncateElement = new Cuttr('.classname', {
    truncate: 'words',
    length: 12,
    readMore: true,
    readMoreBtnSelectorClass: 'cuttr__readmore',
    readMoreBtnAdditionalClasses: 'btn btn-primary',
});

document.getElementsByClassName("cuttr__readmore")[0].addEventListener("click",
  function(){ 
    document.getElementsByClassName("cuttr__readmore")[0].classList.toggle("btn-success");
  });
```

I created a [jsfiddle](https://jsfiddle.net/7j63pm12/4/) for this example.

Hope this helps.
Best regards

**Comment by d-e-v-s-k** [comment]
As callback functions were implemented with version 1.4.0, you can also use the following method:

``` javascript
let truncateElement = new Cuttr('.classname', {
    truncate: 'words',
    length: 12,
    readMore: true,
    readMoreBtnSelectorClass: 'cuttr__readmore',
    readMoreBtnAdditionalClasses: 'btn btn-primary',
    afterTruncate: function(){
        document.querySelector(".cuttr__readmore").classList.add("deep-orange","darken-1");
      },
    afterExpand: function(){
      	document.querySelector(".cuttr__readmore").classList.remove("deep-orange","darken-1");
    }
});
```

See another [fiddle](https://jsfiddle.net/k78vf90j/1/) for example.
